source/Tentacle.sln

```diff
@@ -7,8 +7,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 	ProjectSection(SolutionItems) = preProject
 		..\.gitattributes = ..\.gitattributes
 		..\.gitignore = ..\.gitignore
-		..\build.cake = ..\build.cake
```

Comment:
I'm guess these lines are no longer required and are left over from previous changes?

Reply:
Yeah, they stuck out as problems when I was using the Solution Items section to access the other files. Just some cleanup.